hack/generate/gen-test-framework.sh

```diff
@@ -14,17 +14,10 @@ DOCKERFILE="build/Dockerfile"
 # Go inside of the mock data test project
 cd test/test-framework
 
-# Create the required files in the mock test/test-framework just to allow run from it the oprator-sdk commands
-echo > $DOCKERFILE
-go mod init test-framework
-
-# Remove files that are required and created just to exec the commands.
-trap_add 'rm -rf $DOCKERFILE go.mod go.sum' EXIT
-
```

Comment:
Does `go.sum` need to be checked in? If so, maybe we should run `go mod tidy` in this script to make it easier to keep up-to-date?

Reply:
tidy is already run in `sanity-check.sh`

Reply:
I'm not sure that helps us. Does `go mod tidy` run recursively against all other modules (e.g. `test/test-framework`) in the repo? I don't think it does.

If we include it here, a developer can run `make gen-test-framework` after making changes in the test framework to pick up all of the necessary changes. `go.mod` and `go.sum` will get updated along with the other generated files.

Reply:
Ah I see what you mean. It does not. I will add it here.